doc/manual/src/command-ref/nix-env/delete-generations.md

```diff
@@ -9,14 +9,39 @@
 # Description
 
 This operation deletes the specified generations of the current profile.
-The generations can be a list of generation numbers, the special value
-`old` to delete all non-current generations, a value such as `30d` to
-delete all generations older than the specified number of days (except
-for the generation that was active at that point in time), or a value
-such as `+5` to keep the last `5` generations ignoring any newer than
-current, e.g., if `30` is the current generation `+5` will delete
-generation `25` and all older generations. Periodically deleting old
-generations is important to make garbage collection effective.
+
+*generations* can be a one of the following:
+
+- <span id="generations-list">`<number>...`</span>:\
+  A list of generation numbers, each one a separate command-line argument.
+
+  Delete exactly the profile generations given by their generation number.
+  Deleting the current generation is not allowed.
+
+- The special value <span id="generations-old">`old`</span>
+
+  Delete all generations older than the current one.
+
+- <span id="generations-days">`<days>d`</span>:\
+  The last *days* days
+
+  *Example*: `30d`
+
+  Delete all generations older than *days* days.
+  The generation that was active at that point in time is excluded, and will not be deleted.
```

Comment:
I really don't know what this is supposed to mean.

Reply:
This is taking the parenthetical from above and making it a standalone sentence --- I am not sure exactly either!

We can figure it out and make it clear in a follow-up PR. For now, it is at least no worse than before.

Reply:
> This command will delete all generations older than 30 days, except for the generation that was active 30 days ago (if it currently exists).

This is my interpretation

Reply:
Maybe to slightly clarify then...

```suggestion
  The generation that was active at that point in the past is excluded, and will not be deleted.
```

Reply:
Yeah I like it. I will go read the code now to double check too.